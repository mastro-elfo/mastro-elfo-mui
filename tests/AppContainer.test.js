import React from "react";
import renderer from "react-test-renderer";
import { AppContainer } from "../";

it("renders AbsoluteCircularProgress", () => {
  const tree = renderer.create(<AppContainer />).toJSON();
  expect(tree).toMatchSnapshot();
});
