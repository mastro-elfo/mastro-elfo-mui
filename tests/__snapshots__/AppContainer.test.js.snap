// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AbsoluteCircularProgress 1`] = `null`;
